app/src/main/java/com/cp/tasky/auth/login/presentation/LoginScreen.kt

```diff
@@ -31,10 +31,25 @@ import com.cp.tasky.ui.theme.LARGE_16_DP
 import com.cp.tasky.ui.theme.MAJOR_EXTRA_LARGE_64_DP
 import com.cp.tasky.ui.theme.MINOR_EXTRA_LARGE_48_DP
 
+@Composable
+fun LoginScreenRoot(
+    modifier: Modifier = Modifier,
+    loginViewModel: LoginViewModel = hiltViewModel()
+) {
+
+    LoginScreen(
+        modifier = modifier,
+        viewState = loginViewModel.loginScreenState,
+        onEvents = loginViewModel::onEvents
+    )
+}
+
+
 @Composable
 fun LoginScreen(
     modifier: Modifier = Modifier,
-    viewModel: LoginViewModel = hiltViewModel()
+    viewState: LoginScreenState,
+    onEvents: (LoginScreenEvents) -> Unit
```

Comment:
I would usually keep my class names in singular, since you need to look at what one instance of this class represents (and that represents a single event). Unless you have a list or so, then the single list instance corresponds to multiple items.